index.ts

```diff
@@ -167,6 +243,32 @@ export class BalancerPoolsAPI extends Stack {
       projectionType: ProjectionType.ALL,
     });
 
+    const byAprReadScaling = new ScalableTarget(this, 'byAprReadScaling', {
```

Comment:
scaling instance creation is very similar for all indexes. may be we can put it in some function, to make it more DRY

Reply:
Yea true. Refactored this into a function.